src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/BlobDispatcherTask.java

```diff
@@ -33,7 +33,8 @@ public BlobDispatcherTask(
         this.containers = serviceConfiguration.getEnabledSourceContainers();
     }
 
-    @Scheduled(fixedDelayString = "${scheduling.task.scan.delay}")
+    //@Scheduled(fixedDelayString = "${scheduling.task.scan.delay}")
+    @Scheduled(fixedDelayString = "50")
```

Comment:
How about throwing the exception right after logging the start of the job, so that nothing else happens in this job?

Reply:
@lgonczar In this case we have multiple exceptions per one job run. Let's have a look at how existing code behaves, after some time I will add throwing exception immediately.